packages/genesys-spark-components/src/components/stable/gux-popover/gux-popover.tsx

```diff
@@ -215,17 +218,43 @@ export class GuxPopover {
   componentDidLoad(): void {
     if (this.isOpen) {
       this.runUpdatePosition();
+      this.focusFirstSlottedElement();
     }
   }
 
   componentDidUpdate(): void {
+    this.lastActiveElement = document.activeElement as HTMLElement;
+
     if (this.isOpen) {
+      this.focusFirstSlottedElement();
       this.runUpdatePosition();
     } else if (this.cleanupUpdatePosition) {
       this.cleanupUpdatePosition();
     }
   }
 
+  private focusFirstSlottedElement(): void {
+    this.contentElement.focus();
+
+    // TODO: should we dynamically focus first focusable elemtn in the slotted content?
+    //
+    // // Focus the first focusable element in the slotted content
+    // const contentElementChildren = this.getRootChildren(this.contentElement);
+    // if (contentElementChildren?.length) {
+    //   const firstChild = this.getRootChildren(this.contentElement)[0] as HTMLElement;
+    //   const firstSlottedInput = firstChild.querySelector(
+    //     'gux-button gux-dropdown'
```

Comment:
Problem with the code I have here if we want to dynamically focus the first focusable element is that what I have on line 246 is not scalable since we'd have to maintain a whitelist of focusable gux components. Maybe instead we could search for a raw input or button tag if we go with this approach.